packages/asc-ui/src/components/AmsterdamLogo/AmsterdamLogo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { ReactComponent as LogoShort } from '@datapunt/asc-assets/lib/Icons/LogoShort.svg'
+import { ReactComponent as LogoTall } from '@datapunt/asc-assets/lib/Icons/LogoTall.svg'
+import AmsterdamLogoStyle from '../../styles/components/AmsterdamLogo/AmsterdamLogoStyle'
+
+type Props = {
+  tall?: boolean
+  href?: string
+}
+
+const AmsterdamLogo: React.FC<Props> = ({ tall, ...otherProps }) => (
+  <AmsterdamLogoStyle {...{ tall }} {...otherProps}>
+    <LogoShort />
```

Comment:
this construction gives the impression that both logo's are to be renderd here. Could be this be rendered depending on the tall property?